recipes/unicorn/all/conandata.yml

```diff
@@ -1,14 +1,19 @@
 sources:
-  "1.0.3":
-    url: "https://github.com/unicorn-engine/unicorn/archive/refs/tags/1.0.3.tar.gz"
-    sha256: "64fba177dec64baf3f11c046fbb70e91483e029793ec6a3e43b028ef14dc0d65"
+  "2.0.1":
+    url: "https://github.com/unicorn-engine/unicorn/archive/refs/tags/2.0.1.tar.gz"
+    sha256: "0c1586f6b079e705d760403141db0ea65d0e22791cf0f43f38172d49497923fd"
   "2.0.0":
     url: "https://github.com/unicorn-engine/unicorn/archive/refs/tags/2.0.0.tar.gz"
     sha256: "67b445c760e2bbac663e8c8bc410e43311c7fc92df4dfa8d90e06a021d07f634"
-patches:
   "1.0.3":
-    - patch_file: "patches/1.0.3-0001-cmake-crossbuild-support.patch"
-      base_path: "source_subfolder"
+    url: "https://github.com/unicorn-engine/unicorn/archive/refs/tags/1.0.3.tar.gz"
+    sha256: "64fba177dec64baf3f11c046fbb70e91483e029793ec6a3e43b028ef14dc0d65"
+patches:
+  "2.0.1":
+    - patch_file: "patches/2.0.1-0001-cmake-msvc-support.patch"
+    - patch_file: "patches/2.0.1-0002-cmake-msvc-runtime.patch"
```

Comment:
Please add patch fields for new patches 🙏

Reply:
Thanks, now I found a whole wiki section about it. Will do for future PRs.

Reply:
#14015 will make it hopefully easier